Backpack/src/main/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheet.kt

```diff
@@ -16,28 +16,20 @@
  * limitations under the License.
  */
 
-package net.skyscanner.backpack.demo.components
+package net.skyscanner.backpack.bottomsheet
 
-import android.annotation.SuppressLint
 import android.content.Context
 import android.util.AttributeSet
-import android.view.MotionEvent
-import androidx.core.widget.NestedScrollView
+import androidx.cardview.widget.CardView
+import net.skyscanner.backpack.R
 
-class LockableNestedScrollView @JvmOverloads constructor(
+open class BpkBottomSheet @JvmOverloads constructor(
```

Comment:
Not sure why git considers this as a file change, it's file deletion and file addition :)

Reply:
It does that sometimes, it's weird 🤷